README.md

```diff
@@ -58,8 +58,35 @@ Before opening a PR, check the service renders successfully at http://localhost:
 and run all the tests locally as described below.
 
 ## Adding to your service
+
+### Users of the play-ui or play-frontend-hmrc (versions 8.12.0/0.19.0 or above)
```

Comment:
Maybe 
### Users of the play-ui (version 8.12.0 or above) or play-frontend-hmrc (version 0.19.0 or above)